systemd/leeloo.json

```diff
@@ -0,0 +1,57 @@
+{
+  "transport": {
+    "libp2p": {
+      "bootstrapAddrs": [
+        "/dns/spire-bootstrap1.makerops.services/tcp/8000/p2p/12D3KooWRfYU5FaY9SmJcRD5Ku7c1XMBRqV6oM4nsnGQ1QRakSJi",
+        "/dns/spire-bootstrap2.makerops.services/tcp/8000/p2p/12D3KooWBGqjW4LuHUoYZUhbWW1PnDVRUvUEpc4qgWE3Yg9z1MoR"
+      ]
+    }
+  },
+  "feeds": [
+    "0xDA1d2961Da837891f43235FddF66BAD26f41368b",
+    "0x4b0E327C08e23dD08cb87Ec994915a5375619aa2",
+    "0x75ef8432566A79C86BBF207A47df3963B8Cf0753",
+    "0x83e23C207a67a9f9cB680ce84869B91473403e7d",
+    "0xFbaF3a7eB4Ec2962bd1847687E56aAEE855F5D00",
+    "0xfeEd00AA3F0845AFE52Df9ECFE372549B74C69D2",
+    "0x71eCFF5261bAA115dcB1D9335c88678324b8A987",
+    "0x8ff6a38A1CD6a42cAac45F08eB0c802253f68dfD",
+    "0x16655369Eb59F3e1cAFBCfAC6D3Dd4001328f747",
+    "0xD09506dAC64aaA718b45346a032F934602e29cca",
+    "0xc00584B271F378A0169dd9e5b165c0945B4fE498",
+    "0x60da93D9903cb7d3eD450D4F81D402f7C4F71dd9",
+    "0xa580BBCB1Cee2BCec4De2Ea870D20a12A964819e",
+    "0xD27Fa2361bC2CfB9A591fb289244C538E190684B",
+    "0x8de9c5F1AC1D4d02bbfC25fD178f5DAA4D5B26dC",
+    "0xE6367a7Da2b20ecB94A25Ef06F3b551baB2682e6",
+    "0xA8EB82456ed9bAE55841529888cDE9152468635A",
+    "0x130431b4560Cd1d74A990AE86C337a33171FF3c6",
+    "0x8aFBD9c3D794eD8DF903b3468f4c4Ea85be953FB",
+    "0xd94BBe83b4a68940839cD151478852d16B3eF891",
+    "0xC9508E9E3Ccf319F5333A5B8c825418ABeC688BA",
+    "0x77EB6CF8d732fe4D92c427fCdd83142DB3B742f7",
+    "0x3CB645a8f10Fb7B0721eaBaE958F77a878441Cb9",
+    "0x4f95d9B4D842B2E2B1d1AC3f2Cf548B93Fd77c67",
+    "0xaC8519b3495d8A3E3E44c041521cF7aC3f8F63B3",
+    "0xd72BA9402E9f3Ff01959D6c841DDD13615FFff42"
+  ],
+  "ethereum": {
+    "from": "",
+    "keystore": "",
+    "password": ""
+  },
+  "leeloo": {
+    "listeners": {
+      "wormhole": [
+        {
+          "rpc": [
+          ],
+          "interval": 60,
+          "blocksBehind": [30, 5760, 11520, 17280, 23040, 28800, 34560],
+          "maxBlocks": 1000,
+          "addresses": ["0x20265780907778b4d0e9431c8ba5c7f152707f1d"]
```

Comment:
Does it make sense to change it to the current test contract on konvan-optimism?

```
            {
              "rpc": [],
              "interval": 30,
              "blocksBehind": [10, 500, 1000, 2500, 5000, 10000, 20000, 30000, 40000, 50000, 60000],
              "maxBlocks": 1000,
              "addresses": [
                "0x45440Ae4988965A4cD94651E715fC9A04e62Fb41"
              ]
            }
```